svf/lib/MemoryModel/AccessPath.cpp

```diff
@@ -112,6 +112,9 @@ APOffset AccessPath::computeConstantByteOffset() const
         ///     i = 0, type: [10 x i8]*, PtrType, op = 0
         ///     i = 1, type: [10 x i8], ArrType, op = 8
         const SVFType* type = offsetVarAndGepTypePairs[i].second;
+        /// if offsetVarAndGepTypePairs[i].second is nullptr, it means
+        /// this GepStmt comes from external API, assert
```

Comment:
Add more comments about this from SVFIRBuilder.cpp:getBaseTypeAndFlatternedFields